packages/chrome/src/elements/header/HeaderItem.tsx

```diff
@@ -18,13 +18,13 @@ import { PRODUCTS } from '../../utils/types';
 interface IHeadItemProps
   extends IStyledBaseHeaderItemProps,
     IStyledLogoHeaderItemProps,
-    HTMLAttributes<HTMLElement> {
+    ButtonHTMLAttributes<HTMLElement> {
   /** Applies header logo styles to the item */
   hasLogo?: boolean;
 }
 
 /**
- * Accepts all `<button>` props
+ * @extends ButtonHTMLAttributes<HTMLElement>
```

Comment:
Unfortunately, the previous doc was inaccurate. This refs to a `<div>` and should be `HTMLAtttributes<HTMLDivElement>`.